.github/renovate.json

```
@@ -0,0 +1,20 @@
+{
+    "extends": ["config:base", ":pinOnlyDevDependencies"],
```

WDYT about also opting into replacements? I've found it to be useful for discovering package deprecations/renames.

```suggestion
    "extends": ["config:base", ":pinOnlyDevDependencies", "replacements:all"],
```